build.mjs

```diff
@@ -128,6 +128,9 @@ async function generateVendorScript(componentSetPath) {
 
         // Support script for parallax effect hero components
         path.join(scriptsDir, 'heroes.js'),
+
+        // Support script for chart provider integration in chart components
+        path.join(scriptsDir, 'chart.js'),
```

Comment:
How would we support multiple chart providers in one component set?

Reply:
we can export new ones too in `chart.js`

Reply:
Or you think it would be better to rename `chart.js` to `infogram.js` so each provider would be added through a separate script?
This improvements are can be done for M2, but in order to not forget about it I would like to create a ticket once it is clear what is best to do.

Reply:
> we can export new ones too in chart.js

This would cause all providers to be loaded as soon as we need any of them. Keeping a bootstrapping js file per provider helps preventing that.